README.md

```diff
@@ -68,12 +68,11 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- Using one of our [executable installers](https://github.com/spyder-ide/spyder/releases)
```

Comment:
Ok, thanks a lot for that! But until we have the installers ready, please do these modifications so we can merge this PR.

Then you can restore this text again.